### PR TITLE
847219 : Image rendering issue occurs while converting Html to PDF using WebKit.

### PR DESCRIPTION
**Pull request by Sri-hari-haran-g**
Hi All,

I have added the troubleshooting for "Image rendering issue occurs while converting Html to PDF using WebKit" topic. Kindly review this.

Regards,
Srihariharan Gopalakrishnan

### CODE REVIEW

File: File-Formats/PDF/Convert-HTML-To-PDF/troubleshooting.md [hunk 1]
```diff
@@ -452,6 +452,45 @@ The runtime files, or blink binaries, will be copied into a bin or published fol
 
 By including the <b><ExcludeAssets>native</ExcludeAssets></b> option in the package reference of the csproj file, you can exclude the runtime files or blink binaries from being copied into the bin or publish folder while building and publishing the application. But you need to place the BlinkBinaries in the server disk and set the BlinkPath in the BlinkConverterSettings to perform the conversion. 
 
+## Preservation issues occurs while converting HTML(HTTPS sites) to PDF using WebKit rendering engine in local machine/Azure App service
```
Comment (SowmiyaLoganathan) [inline File-Formats/PDF/Convert-HTML-To-PDF/troubleshooting.md:455]: Use this title "Image rendering issue occurs while converting Html to PDF using WebKit"

File: File-Formats/PDF/Convert-HTML-To-PDF/troubleshooting.md [hunk 2]
```diff
@@ -452,6 +452,45 @@ The runtime files, or blink binaries, will be copied into a bin or published fol
...
+
+<table>
+<th style="font-size:14px" width="100px">Issue
+</th>
+<th style="font-size:14px">Preservation issues occurs while converting HTML(HTTPS sites) to PDF using WebKit rendering engine in local machine/Azure App Service.
```
Comment (SowmiyaLoganathan) [inline File-Formats/PDF/Convert-HTML-To-PDF/troubleshooting.md:460]: Image rendering issue occurs while converting HTML(HTTPS sites) to PDF using Webkit rendering engine in location machine/Azure App Service.

File: File-Formats/PDF/Convert-HTML-To-PDF/troubleshooting.md [hunk 3]
```diff
@@ -452,6 +452,45 @@ The runtime files, or blink binaries, will be copied into a bin or published fol
...
+<table>
+<th style="font-size:14px" width="100px">Issue
+</th>
+<th style="font-size:14px">Preservation issues occurs while converting HTML(HTTPS sites) to PDF using WebKit rendering engine in local machine/Azure App Service. 
+</th>
+
+<tr>
+<th style="font-size:14px" width="100px">Reason
+</th>
+<td>The issue occurs due to missing of OPENSSL assemblies in your machine.
```
Comment (SowmiyaLoganathan) [inline File-Formats/PDF/Convert-HTML-To-PDF/troubleshooting.md:466]: When the OPENSSL assemblies are missing in your machine.

File: File-Formats/PDF/Convert-HTML-To-PDF/troubleshooting.md [hunk 4]
```diff
@@ -452,6 +452,45 @@ The runtime files, or blink binaries, will be copied into a bin or published fol
...
+</td>
+</tr>
+
+<tr>
+<th style="font-size:14px" width="100px">Solution
+</th>
+<td>
+Our WebKit rendering engine requires OPENSSL libraries to be installed in the machine where the conversion takes place.
```
Comment (SowmiyaLoganathan) [inline File-Formats/PDF/Convert-HTML-To-PDF/troubleshooting.md:474]: Our WebKit rendering engine requires OPENSSL assemblies for accessing the resources from HTTPS URL. If you are accessing any resources from HTTPS sites, then kindly include these below assemblies in your project,
ExcludeAssets<br><br>

You can get the OPENSSL assemblies from the below link, 
<a href="https://www.syncfusion.com/downloads/support/directtrac/general/ze/OPENSSL-798051511">https://www.syncfusion.com/downloads/support/directtrac/general/ze/OPENSSL-798051511</a>

File: File-Formats/PDF/Convert-HTML-To-PDF/troubleshooting.md [hunk 5]
```diff
@@ -452,6 +452,35 @@ The runtime files, or blink binaries, will be copied into a bin or published fol
 
 By including the <b><ExcludeAssets>native</ExcludeAssets></b> option in the package reference of the csproj file, you can exclude the runtime files or blink binaries from being copied into the bin or publish folder while building and publishing the application. But you need to place the BlinkBinaries in the server disk and set the BlinkPath in the BlinkConverterSettings to perform the conversion. 
 
+## Image rendering issue occurs while converting Html to PDF using WebKit
```
Comment (SowmiyaLoganathan) [inline File-Formats/PDF/Convert-HTML-To-PDF/troubleshooting.md:455]: Html should be "HTML"

**Review by PrakashViswanath — APPROVED** [review]
The changes are fine.

**Pull request merged**